koku/api/report/test/ocp/test_views.py

```diff
@@ -45,6 +45,11 @@ def string_to_date(dt):
     return datetime.datetime.strptime(dt, "%Y-%m-%d").date()
 
 
+def get_distributed_cost(dikt):
+    "given a dictionary with a nested vlaues return distributed cost"
+    return dikt.get("values")[0].get("cost", {}).get("distributed", {}).get("value")
```

Comment:
nitpick:

There's a possibility for `TypeError: 'NoneType' object is not subscriptable`  if `dikt.get("values")` were to return 
 `None`